consumer_kafka/src/main/java/com/example/Application.java

```diff
@@ -79,19 +61,14 @@ public void listen(Foo2 foo) {
 		this.storedFoo = foo;
 	}
 
-	@KafkaListener(id = "dltGroup", topics = "topic1.DLT")
-	public void dltListen(String in) {
-		logger.info("Received from DLT: " + in);
+	@KafkaListener(id = "fooMessageGroup", topics = "topic1")
+	public void listenFooMessage(Message<Foo2> fooMessage) {
```

Comment:
This listens for the "Message" of Foo so that we can inspect the Kafka message keys.